src/opm/material/fluidmatrixinteractions/EclMaterialLawManager.cpp

```diff
@@ -81,6 +81,22 @@ initFromState(const EclipseState& eclState)
                 stoneEtas_.push_back(table.eta);
             }
         }
+        
+        const auto& ppcwmaxTables = tables.getPpcwmax();
+        this->enablePpcwmax_ = !ppcwmaxTables.empty() ? true : false;
```

Comment:
It's slightly more idiomatic to write this as
```C++
this->enablePpcxwmax_ = !ppcwmaxTables.empty();
```
The `.empty()` function already returns a `bool` so you don't need the explicit `true` and `false` values here.

Reply:
Thanks, I changed this.